README.md

```diff
@@ -385,6 +385,18 @@ You will need:
 - `--config-template-path` option to provide a path to a config file.
 - `--config-type` option specify which default template to use to create the config file.
 
+## Cancel pending transaction
+This command will cancel pending transaction.
+
+Please note that a this action is irreversible
+
+```
+open-attestation transaction cancel --nonce <PENDING TRANSACTION NONCE> --gas <GAS FEE> [options]
```

Comment:
gas => gas-price

Reply:
Noted, will change --gas to --gas-price and the description in readme.